WORKSPACE

```diff
@@ -103,16 +103,24 @@ hsm_repos()
 load("//rules:bitstreams.bzl", "bitstreams_repo")
 bitstreams_repo(name = "bitstreams")
 
-# Setup for linking in external test hooks.
-load("//rules:hooks_setup.bzl", "hooks_setup")
+# Setup for linking in external test hooks for both secure/non-secure
+# manufacturer domains.
+load("//rules:hooks_setup.bzl", "hooks_setup", "secure_hooks_setup")
 hooks_setup(
     name = "hooks_setup",
     dummy = "sw/device/tests/closed_source",
 )
+secure_hooks_setup(
+    name = "secure_hooks_setup",
+    dummy = "sw/device/tests/closed_source",
+)
 
-# Declare the external test_hooks repository.
+# Declare the external test_hooks repositories. One for both manufacturer secure
+# and non-secure domains.
 load("@hooks_setup//:repos.bzl", "hooks_repo")
+load("@secure_hooks_setup//:repos.bzl", "hooks_repo")
```

Comment:
need to swap lines #121 and #122 (non-secure repo should be instantiated before declaration of secure domain hooks_repo)

Reply:
ahh, good catch. I updated the bazel rules so order shouldn't matter now